crawl-ref/source/ability.cc

```diff
@@ -79,30 +79,30 @@
 
 enum class abflag
 {
-    NONE           = 0x00000000,
```

Comment:
Changes to whitespace in otherwise-unchanged code should really be in a separate commit.